src/test/java/com/napier/sem/Record_UnitTests.java

```diff
@@ -4,16 +4,14 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-public class Record_UnitTests
-{
+public class Record_UnitTests {
```

Comment:
Could this be moved back on to a new line. Could have been the InteliJ formatting thing again.